account_invoice_rounding/test/test_invoice_rounding.yml

```diff
@@ -17,6 +17,37 @@
     type_tax_use: all
     tax_code_id: tax_case
 -
+  I create a Tax Code for 8%
```

Comment:
It seems 9.0 and master use them:
https://github.com/odoo/odoo/blob/9.0/addons/stock/__openerp__.py#L79
https://github.com/odoo/odoo/blob/master/addons/stock/__openerp__.py#L94